pkg/component/etcd/copybackupstask/copybackupstask.go

```diff
@@ -100,6 +101,10 @@ func (e *etcdCopyBackupsTask) Deploy(ctx context.Context) error {
 	e.task.Spec.SourceStore = e.values.SourceStore
 	e.task.Spec.TargetStore = e.values.TargetStore
 	e.task.Spec.WaitForFinalSnapshot = e.values.WaitForFinalSnapshot
+	e.task.Spec.PodLabels = map[string]string{
+		v1beta1constants.LabelNetworkPolicyToDNS:            v1beta1constants.LabelNetworkPolicyAllowed,
+		v1beta1constants.LabelNetworkPolicyToPublicNetworks: v1beta1constants.LabelNetworkPolicyAllowed,
+	}
```

Comment:
How did this work before/why this is only needed now?

Reply:
This is captured as part of etcd-druid v0.28.0 release notes.
```
EtcdCopyBackupTask introduces a new field spec.PodLabels which should be used to setup additional labels on the pod(s) that are started for etcd-copy-backup operation
```
Previously these labels were hard coded into etcd-druid. We removed the hard coding and made it configurable via `PodLabels`. For other resources network related labels are set in g/g codebase. This change harmonises that and removes hard coding of g/g specific labels in etcd-druid code base.